### PR TITLE
[PD-3884] Include deleted comm recs in comm rec reports

### PR DESCRIPTION
**Pull request by koonsies**
## What's this PR do?
Communication records that have been archived do not currently display in the communication record report.

Display these records, along with an archived date so that the user can tell the record is archived.

## Where should the reviewer start?
PRM or reporting.

## How should this be manually tested?
If you already have archived comm recs, head to reporting and run a report.
If not, delete a contact with comm recs, or a few comm recs individually.
Ensure the reports now contain the deleted comm recs.

## Any background context you want to provide?
There is a small change to schema.py that removes old partner assessment code.

### Separation of Concerns

- [ ] Back end "business logic" is implemented in plain python, Django model classes, and Django forms. (not Django views or other frameworks)
- [ ] Concerns of UI, api, and model are in separate code units.

### Tests

- [ ] Base classes used for python tests don't do too much unnecessary work.

### Migrations

- [ ] Migrations can be reversed.
- [ ] Current production code would continue to work after this migration is applied.
- [ ] Data migrations' forward/reverse functions should be decorated with `@only_on_default`.

### CODE REVIEW

File: myreports/datasources/comm_records.py [hunk 1]
```diff
@@ -212,11 +213,8 @@ def help_contact(self, company, filter_spec, partial):
     def filtered_query_set(self, company, filter_spec):
         """Create a query set with security, safety, and user filters applied.
         """
-        qs_company = ContactRecord.objects.filter(partner__owner=company)
-        qs_live = (
-            qs_company
-            .filter(approval_status__code__iexact=Status.APPROVED)
-            .filter(archived_on__isnull=True))
+        qs_company = ContactRecord.all_objects.filter(partner__owner=company)
+        qs_live = qs_company.filter(approval_status__code__iexact=Status.APPROVED)
```
Comment (koonsies) [inline myreports/datasources/comm_records.py]: Remove the explicit `isnull` directive.

File: myreports/schema.py [hunk 2]
```diff
@@ -7,7 +7,7 @@
     ReportConfiguration, ColumnConfiguration)
 
 from myreports.schema_columns import (
-    cols_contacts, cols_partners, cols_comm_rec, cols_tjd_report, cols_comm_rec_assessment, cols_partner_assessment)
+    cols_contacts, cols_partners, cols_comm_rec, cols_tjd_report, cols_partner_assessment)
```
Comment (koonsies) [inline myreports/schema.py]: This removes some code only used for waffling partner assessments.

File: myreports/schema_columns.py [hunk 3]
```diff
@@ -481,9 +481,6 @@
         label=None,
         description=None,
     ),
-]
-
-cols_comm_rec_assessment = [
```
Comment (koonsies) [inline myreports/schema_columns.py]: This removes some code only used for waffling partner assessments.

**Comment by almathew** [comment]
Test failures are legit.

**Comment by koonsies** [comment]
Closing to fix tests.

**Pull request merged**